### PR TITLE
Prove It QA

### LINKED ISSUE #1455: action page (prove it) - design QA todos
**Desktop**
- [x] Remove colon from copy accompanying sponsor logo
- [x] Change "give us a shout" to be “contact us”
- [x] Text on the bottom line should have the the same color but the secondary link is underlined
- [x] Make sure the last line of text has the same left and right margins as the rest of the body

**Mobile**
- [x] Address multi-line call to action button
- [x] Address huge button size on mobile
- [x] move the sources back up to Step 1

**EDIT:** Removed "buggy photo elements" item from this issue as it was already described in #1455

### PR DESCRIPTION
**Pull request by mmwtsn**
## Changes
- Help footer copy change
- Sponsored header copy cleanup
- Reduces default size of location finder and report back buttons
- Tweaks "Prove It" column widths to prevent buttons from breaking to two lines after font-size increase
- Tweaks campaign navigation width and mobile breakpoint after font-size increase

Resolves #1455

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss [hunk 1]
```diff
@@ -57,9 +56,12 @@
 
   .content {
     margin-bottom: 3rem;
+    text-align: center;
 
     @include media($tablet) {
-      @include span-columns(5);
+      text-align: left;
+
+      @include span-columns(6.31);
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss]: srsly tho why are we doing this

File: lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss [hunk 2]
```diff
@@ -68,7 +70,7 @@
     margin: 2rem 0;
 
     @include media($tablet) {
-      @include span-columns(6);
+      @include span-columns(5.69);
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss]: what have you done

File: lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss (hunk 1 above)
Comment (mmwtsn) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/campaign/_prove.scss]: See "The Danger Zone" at 4:29 PM today.

**Comment by DFurnes** [comment]
Adding notes from offline discussion that just happened:

current appearance

proposed smaller button for "update submission"

We're considering the idea of dropping the button down one size when in "update submission" mode, since it fixes our text wrapping issues & also seems pretty sensible since updating your submission shouldn't be as big a call-to-action as "Submit Your Pic" is.

**Comment by mmwtsn** [comment]
:package: Ready to go.

**Pull request merged**